server/package.json

```diff
@@ -19,7 +19,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v9.1.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-upgrade-twilio",
```

Comment:
Remember to update this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/54 has been merged

Reply:
Done.